src/Daemon/AlarmManager.vala

```diff
@@ -184,13 +184,14 @@ namespace HourglassDaemon {
             bool same_min = alarm_min == now.get_minute ();
 
             if (same_hour && same_min) {
-                //if today, alarm goes off
-                if (same_day && same_month && same_year) {
+                int today = now.get_day_of_week () != 7 ? now.get_day_of_week () : 0;
```

Comment:
Tiny thing but would you tell me why you moved this variable here? I don't see it used in the next `if` sentence so we can keep it as it is, just before it used.

Reply:
My apologies, I should have submitted this when I was less tired. That line should be moved back.